Updaed AA credential doc to include python2

jennyhunter-msft opened the pull request:
-Added description of python2 credential function
-Added example of how to use python2 credential function

azurecla commented:
Hi __@magaal__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

gbowerman commented:
Good catch. Thanks Mike.

magaal commented:
NP - thank szarkos too for narrowing it down :)

jennyhunter-msft committed:
Updaed AA credential doc to include python2 

-Added description of python2 credential function
-Added example of how to use python2 credential function
